index.html

```diff
@@ -5,7 +5,7 @@
     
     <link rel="icon" type="image/svg+xml" href="./logo.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Sprout UI - Chia dApp ProjectsList</title>
+    <title>Sprout UI - Chia dApp UnitsList</title>
```

Comment:
Not sure what this is but it should saw "Climate Action Data Trust"

Reply:
corrected.this was a hold over from sprout-ui